src/core/Group.h

```diff
@@ -39,9 +39,6 @@ class Group : public ModifiableObject
     enum MergeMode
     {
         Default, // Determine merge strategy from parent or fallback (Synchronize)
-        Duplicate, // lossy strategy regarding deletions, duplicate older changes in a new entry
-        KeepLocal, // merge history forcing local as top regardless of age
-        KeepRemote, // merge history forcing remote as top regardless of age
         KeepNewer, // merge history
         Synchronize, // merge history keeping most recent as top entry and applying deletions
```

Comment:
What is actually the difference between these two?

Reply:
[This](https://github.com/keepassxreboot/keepassxc/blob/5fb26d666a67f95c489f10ef4e5c6b881d879772/src/core/Merger.cpp#L517) is the only difference. In a future PR, I think I will remove the `MergeMode` completely, and instead add a `Merger` option called `mergeDeletions` or `synchronizeDeletions`.

Reply:
Agree!